README.md

```diff
@@ -1,4 +1,5 @@
 [![js-standard-style](https://img.shields.io/badge/code%20style-standard-brightgreen.svg)](http://standardjs.com/)
+[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
 
 ## [Webogram](https://web.telegram.org) — Telegram Web App
```

Comment:
Hi. The confirmation code is not a message to my number. Please send the code by voice. Thanks+989359915878